ebpf/tracer.go

```diff
@@ -225,6 +241,9 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 			if t.shouldSkipConnection(&conn) {
 				atomic.AddUint64(&t.skippedConns, 1)
 			} else {
+				// lookup conntrack in for active
+
+				conn.IPTranslation = t.conntracker.GetConntrackEntryForConn(conn.Source, conn.SPort)
```

Comment:
This only grabs it for connections currently in the BPF map (e.g. active connections). 

You'll want to do this above in `initPerfPolling` as well for closed connections.